README.md

```diff
@@ -4,9 +4,11 @@
 ![CI](https://github.com/fabric8-analytics/fabric8-analytics-vscode-extension/workflows/CI/badge.svg?branch=master)
 [![Codecov](https://codecov.io/gh/fabric8-analytics/fabric8-analytics-vscode-extension/branch/master/graph/badge.svg?token=rHIO4KNlJ0)](https://codecov.io/gh/fabric8-analytics/fabric8-analytics-vscode-extension)
 
-Red Hat's Dependency Analytics (RHDA) extension gives you awareness to security concerns within your software supply chain while you build your application.
-The Red Hat Dependency Analytics extension uses the Snyk REST API to query [Snyk's Vulnerability Database](https://snyk.io/product/vulnerability-database/) for the most up-to-date vulnerability information available.
-Snyk uses industry-leading security intelligence by pulling from many data sources to give you exact vulnerability information.
+Red Hat's Dependency Analytics (RHDA) extension gives you awareness to security concerns within your software supply chain while you code your application.
+The Red Hat Dependency Analytics extension uses the following data sources for the most up-to-date vulnerability information available:
+
+- [Snyk's Vulnerability Database](https://snyk.io/product/vulnerability-database/) is an industry-leading security intelligence service. Red Hat Dependency Analytics uses Snyk's REST API to query for the latest vulnerability information available.
+- The [ONGuard](https://github.com/RHEcosystemAppEng/ONguard) service, which integrates the [Open Source Vulnerability (OSV)](https://google.github.io/osv.dev/) and the [National Vulnerability Database (NVD)](https://nvd.nist.gov/) data sources. When giving a set of package URLs (purls) to the ONGuard service, a query to OSV retrieves the associated vulnerability information, and also returns Common Vulnerability and Exposures (CVE) metrics.
```

Comment:
I'd say that this service queries OSV to retrieve vulnerabilities from the given packages and then queries the NVD database to retrieve the public CVE information.
This way you don't give implementation details that might change in the future.

Reply:
Thank you for the feedback!  I've made a slight change based on your suggestion.